backend/src/api/helper.clj

```diff
@@ -49,6 +64,7 @@
    :player-id player-id
    :hand (get-hand game-state player-id)
    :rows (get-rows game-state player-id)
+   :rows-power (get-rows-power game-state player-id)
    :scores (get-scores game-state player-id)
    :winner (get-winner game-state player-id)})
 
```

Comment:
I still think any function below this either should be calculated inside `get-game-as-player` or should be in a different file, with this one being named `api.player-view` or something like that

Reply:
I was not sure about the `api.player-view-functions` but at least it's a bit clearer now.